main/alpine-baselayout/profile

```diff
@@ -1,5 +1,24 @@
+# /etc/profile
+
+appendpath () {
```

Comment:
I think this function is not needed.
Doing
`export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$PATH` would be enough.

Reply:
See the implementaiton on other distros, ArchLinux: https://git.archlinux.org/svntogit/packages.git/tree/trunk/profile?h=packages/filesystem#n6 and CentOS: https://releases.pagure.org/setup/setup-2.12.6.tar.bz2. The reason why they use this approach is to deduplicate the entries in PATH in a safe manner.